services/src/main/java/za/co/sfy/lucid/sams/rest/controller/LearnerInfoController.java

```diff
@@ -0,0 +1,58 @@
+package za.co.sfy.lucid.sams.rest.controller;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import za.co.sfy.lucid.sams.domain.ServiceStatus;
+import za.co.sfy.lucid.sams.domain.exception.LucidSamsExecutionException;
+import za.co.sfy.lucid.sams.rest.service.LearnerInfoService;
+import za.co.sfy.lucid.sams.rest.vo.data.writer.LearnerInfoRequest;
+import za.co.sfy.lucid.sams.rest.vo.data.writer.LearnerInfoResponse;
+
+import javax.validation.Valid;
+
+/**
+ * @author muzim
+ */
+@RestController
+@RequestMapping("Learner-Info")
+public class LearnerInfoController {
+
+    private static final Logger logger = LoggerFactory.getLogger(LearnerInfoController.class);
+    private LearnerInfoService learnerInfoService;
+
+    @Autowired
+    public LearnerInfoController(final LearnerInfoService learnerInfoService) {
+        this.learnerInfoService = learnerInfoService;
+    }
+
+    @PostMapping(
+            value = "/save",
+            consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<LearnerInfoResponse> saveLearnerInfo(@Valid @RequestBody LearnerInfoRequest learnerInfoRequest) {
```

Comment:
The @valid annotation needs to be used in conjunction to annotations on the LearnerInfoRequest object. Otherwise it doesnt validate anything. Please check this on all requests.